pkg/volume/vsphere_volume/vsphere_volume_util.go

```diff
@@ -40,15 +40,22 @@ const (
 	checkSleepDuration = time.Second
 	diskByIDPath       = "/dev/disk/by-id/"
 	diskSCSIPrefix     = "wwn-0x"
-	diskformat         = "diskformat"
-	datastore          = "datastore"
-	StoragePolicyName  = "storagepolicyname"
+	// diskformat parameter is deprecated as of Kubernetes v1.21.0
+	diskformat        = "diskformat"
```

Comment:
I am still somewhat concerned about deprecating support for `diskformat` option. As per - https://www.vmware.com/pdf/vsp_4_thinprov_perf.pdf - for certain workloads performance of zeroedthick disks are 20-30% faster than thinly provisioned disks for VMFS disk types. Now we are saying - user may not have this choice. 

You mentioned that - `diskformat` might eventually get supported via SPBM policy (or I don't know if this is already supported). Do we have a timeline for this? Do we need an issue to track that?

Reply:
>  Do we have a timeline for this? Do we need an issue to track that?

The feature will be available in the future vSphere release, we do not have the timeline.

> Do we need an issue to track that?

If you can create an issue on the vSphere CSI driver repository, it will help.

Reply:
why csi does not support diskformat?

Reply:
@jingxu97 
vSphere CSI Driver is using [CNS CreateVolume API](https://vdc-download.vmware.com/vmwb-repository/dcr-public/8ed923df-bad4-49b3-b677-45bca5326e85/d2d90bb6-d1b3-4266-8ce5-443680187a9a/vim.cns.VolumeCreateSpec.html) which does not have the capability to supply the "diskformat" parameter while creating volume.